src/server/game/Entities/Creature/GossipDef.cpp

```diff
@@ -23,6 +24,112 @@ GossipMenu::~GossipMenu()
     ClearMenu();
 }
 
+string GossipMenu::GetGossipOptionTextFromDB(uint64 gossipOptionTextID)
```

Comment:
Sorry but I don't think that retrieving these strings everytime from the database everytime you open that gossip menu would be a nice idea.

You should preload them in memory at server startup if you really want to offer an alternative method to load them from acore_strings instead of gossip_menu/npc_text